src/Tokenizer/Transformers.php

```diff
@@ -47,7 +47,7 @@ private function __construct()
     /**
      * @return Transformers
      */
-    public static function create()
+    public static function createSingleton()
```

Comment:
What is the motivation behind rename? Not sure the new one is anyhow better.

Reply:
to unify with `FileReader::createSingleton`

Reply:
Then why not renaming in the `FileReader`? - I believe in OS `create` is most common way for creating class instance.

Reply:
we used `create` for creating an instance and `createSingleton` for creating singleton instance <- better visibility of hacky instance creation